### PR TITLE
Release notes 6.0.0 preview 2

### PR DESCRIPTION
**Pull request by mdaigle**

### CODE REVIEW

File: release-notes/6.0/6.0.0-preview2.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+### Added
+- Added a dependency on System.Text.Json 8.0.5 for .NET 8+ and 6.0.10 for other versions [#2921](https://github.com/dotnet/SqlClient/pull/2921)
+- Added support for JSON datatype [#2916](https://github.com/dotnet/SqlClient/pull/2916), [#2892](https://github.com/dotnet/SqlClient/pull/2892), [#2891](https://github.com/dotnet/SqlClient/pull/2891), [#2880](https://github.com/dotnet/SqlClient/pull/2880), [#2882](https://github.com/dotnet/SqlClient/pull/2882)
+- Added readme to nuget package [#2826](https://github.com/dotnet/SqlClient/pull/2826)
+
+
+### Fixed
+- Fixed scale serialization when explicitly set to 0 [#2411](https://github.com/dotnet/SqlClient/pull/2411)
+- Fixed build issues with common project [#2907](https://github.com/dotnet/SqlClient/pull/2907)
```
Comment (cheenamalhotra) [inline release-notes/6.0/6.0.0-preview2.md:24]: This doesn't need to be called out.

**Review by cheenamalhotra — APPROVED** [review]

**Comment by ErikEJ** [comment]
How about the related index files? Or this is only the base?

File: release-notes/6.0/README.md [hunk 2]
```diff
@@ -5,3 +5,4 @@ The following Microsoft.Data.SqlClient 6.0 preview releases have been shipped:
 | Release Date | Version | Notes |
 | :-- | :-- | :--: |
 | 2024-08-27 | 6.0.0-preview1.24240.8 | [release notes](6.0.0-preview1.md) |
+| 2024-10-25 | 6.0.0-preview2 | [release notes](6.0.0-preview2.md) |
```
Comment (mdaigle) [inline release-notes/6.0/README.md:8]: @ErikEJ is this the spot you were mentioning for the index files? Or are there others you know of as well?

Comment (ErikEJ) [reply release-notes/6.0/README.md:8]: There is a higher level file, but not sure if that usually includes previews?

Comment (mdaigle) [reply release-notes/6.0/README.md:8]: Got it, updated that file.

**Review by David-Engel — APPROVED** [review]

**Review by cheenamalhotra — APPROVED** [review]

**Pull request merged**